dq-game/src/game_types/player.py

```diff
@@ -20,3 +20,14 @@ def __init__(self, name):
     def add_points(self, points):
         self.differential = points
         self.points += points
+
+    def double_differential(self):
+        self.points += self.differential
```

Comment:
Don't you need just to duplicate the differential ?

Reply:
No, for the same reason we sum the points in add_points()